src/dos/dos_programs.cpp

```diff
@@ -197,7 +197,7 @@ class MOUNT : public Program {
 		}
 
 		if (cmd->FindExist("-cd", false)) {
-   			WriteOut(MSG_Get("PROGRAM_MOUNT_NO_CDROM_SUPPORT"));
+   			WriteOut(MSG_Get("PROGRAM_MOUNT_NO_OPTION"), "-cd");
```

Comment:
Much more helpful than the original (which didn't explicitly say which of possibly-multiple options, was the problem)